libsession/src/main/java/org/session/libsession/messaging/utilities/UpdateMessageData.kt

```diff
@@ -0,0 +1,47 @@
+package org.session.libsession.messaging.utilities
+
+import com.fasterxml.jackson.core.JsonParseException
+import org.session.libsignal.service.api.messages.SignalServiceGroup
+import org.session.libsignal.utilities.JsonUtil
+import org.session.libsignal.utilities.logging.Log
+import java.util.*
+
+// class used to save update messages details
+class UpdateMessageData () {
+
+    var type: SignalServiceGroup.Type = SignalServiceGroup.Type.UNKNOWN
+    var groupName: String? = null
+    var updatedMembers: Collection<String> = Collections.emptyList()
```

Comment:
These 3 lines seem like a prime candidate for a sealed class!

Reply:
Yeah gotcha, I forgot about sealed classes, but that's definitely going to improve that class.
I added it. It was a bit tricky around json serialisation, but I figured it out.